### PR TITLE
bug with model loading in R

### PR DESCRIPTION
**Pull request by hoaphumanoid**
There is no way in R to load the data starting in a specific iteration, because inside `mx.model.FeedForward.create` the training function it always starts in 1, instead in a variable. A solution could be to add a variable called `begin.round`.

### CODE REVIEW

**Comment by hoaphumanoid** [comment]
Travis breaks in the roxigen:
```
Error in parse(n = -1, file = file, srcfile = NULL, keep.source = FALSE) : 
  393:10: unexpected symbol
392: function(symbol, X, y=NULL, ctx=NULL, begin.round=1
393:          num.round
             
Calls: <Anonymous> ... parse_package -> load_code -> lapply -> FUN -> parse
Execution halted
make: *** [roxygen] Error 1
```
Did I forget to add something?

File: R-package/R/model.R [hunk 1]
```diff
@@ -387,7 +389,7 @@ mx.model.select.layout.predict <- function(X, model) {
 #' @export
 
 mx.model.FeedForward.create <-
-function(symbol, X, y=NULL, ctx=NULL,
+function(symbol, X, y=NULL, ctx=NULL, begin.round=1
```
Comment (thirdwing) [inline R-package/R/model.R]: you missed a comma at end of this line

**Comment by thirdwing** [comment]
@hoaphumanoid You missed a comma. Please update the PR and let's see the testing result.

**Comment by thirdwing** [comment]
Can you merge the latest changes from master into this branch?

**Pull request merged**